### PR TITLE
Refactor make image

### PR DESCRIPTION
**Pull request by aweris**
Solves issue https://github.com/prometheus-operator/prometheus-operator/issues/3761

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,14 +168,6 @@ kubectl delete --ignore-not-found customresourcedefinitions \
 
 `make test-unit`
 
-#### Running *end-to-end* tests on local minikube cluster:
```
Comment (paulfantom) [inline README.md:171]: I don't think we should remove those instructions as those are still valied. Maybe let's just change order and put `kind` info before minikube one, wdyt?

**Review by paulfantom — COMMENTED** [review]
lgtm, just one nit

File: README.md (hunk 1 above)
Comment (aweris) [reply README.md:171]: Makes sense.

**Comment by paulfantom** [comment]
Thanks! :100:

**Pull request merged**